.jenkins/Jenkinsfile-cryptography-wheel-builder

```diff
@@ -0,0 +1,128 @@
+def configs = [
+    [
+        label: 'windows',
+        versions: ['py26', 'py27', 'py33', 'py34', 'py35', 'py36'],
+    ],
+    [
+        label: 'windows64',
+        versions: ['py26', 'py27', 'py33', 'py34', 'py35', 'py36'],
+    ],
+    [
+        label: 'sierra',
+        versions: ['py27u', 'py27', 'py34', 'py35', 'py36'],
+    ],
+]
+
+
+def build(version, label) {
+    try {
+        timeout(time: 30, unit: 'MINUTES') {
+            if (label.contains("windows")) {
+                def pythonPath = [
+                    py26: "C:\\Python26\\python.exe",
+                    py27: "C:\\Python27\\python.exe",
+                    py33: "C:\\Python33\\python.exe",
+                    py34: "C:\\Python34\\python.exe",
+                    py35: "C:\\Python35\\python.exe",
+                    py36: "C:\\Python36\\python.exe"
+                ]
+                if (version == "py35" || version == "py36") {
+                    opensslPaths = [
+                        "windows": [
+                            "include": "C:\\OpenSSL-Win32-2015\\include",
+                            "lib": "C:\\OpenSSL-Win32-2015\\lib"
+                        ],
+                        "windows64": [
+                            "include": "C:\\OpenSSL-Win64-2015\\include",
+                            "lib": "C:\\OpenSSL-Win64-2015\\lib"
+                        ]
+                    ]
+                } else {
+                    opensslPaths = [
+                        "windows": [
+                            "include": "C:\\OpenSSL-Win32-2010\\include",
+                            "lib": "C:\\OpenSSL-Win32-2010\\lib"
+                        ],
+                        "windows64": [
+                            "include": "C:\\OpenSSL-Win64-2010\\include",
+                            "lib": "C:\\OpenSSL-Win64-2010\\lib"
+                        ]
+                    ]
+                }
+                bat """
+                    @set PATH="C:\\Python27";"C:\\Python27\\Scripts";%PATH%
+                    @set PYTHON="${pythonPath[version]}"
+
+                    @set INCLUDE="${opensslPaths[label]['include']}";%INCLUDE%
+                    @set LIB="${opensslPaths[label]['lib']}";%LIB%
+                    virtualenv -p %PYTHON% .release
+                    call .release\\Scripts\\activate
+                    pip install wheel virtualenv
+                    pip wheel cryptography --wheel-dir=wheelhouse --no-binary cryptography
+                    for %%x in (wheelhouse\\*.whl) do (
+                       pip install %%x
+                    )
```

Comment:
Why is this not `pip install -f wheelhosue cryptography --no-index` like macOS uses?